base_layer/mmr/src/sparse_merkle_tree/mod.rs

```diff
@@ -25,12 +25,8 @@
 //! you will notice that they the first two diverge at the first bit, while the first and last pairs differ at the
 //! fourth bit. This results in a SMT that looks like this:
 //!
-//! .           ┌──────┐
-//!       ┌─────┤ root ├─────┐
-//!       │     └──────┘     │
-//!      ┌┴┐                ┌┴┐1
-//!   ┌──┤ ├──┐          ┌──┤ ├───┐
-//!   │  └─┘  │          │  └─┘   │
+//! . ┌──────┐ ┌─────┤ root ├─────┐ │     └──────┘     │ ┌┴┐                ┌┴┐1 ┌──┤ ├──┐          ┌──┤ ├───┐ │  └─┘  │
+//!   │  └─┘   │
```

Comment:
this is wrong?

Reply:
Yip, it is the `.` in `//! .` that made the issue with cargo format.